src/components/artwork/contact.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import TextLink from '../text_link'
+
+export class ArtworkContact extends React.Component<RelayProps, null> {
+
+  contactLine() {
+    const artwork = this.props.artwork
+    if (artwork.sale && artwork.sale.is_auction) {
+      return this.auctionLine()
+    } else if (artwork.is_inquireable) {
+      return this.contactPartnerLine()
+    }
+  }
+
+  auctionLine() {
+    const artwork = this.props.artwork
+    if (artwork.sale.is_live_open) {
+      return (
+        <TextLink href={artwork.href}>Enter Live Auction</TextLink>
+      )
+    } else if (artwork.sale.is_open) {
+      const sa = artwork.sale_artwork
+      const bids = sa.counts.bidder_positions
+      if (bids > 0) {
+        const s = bids > 1 ? 's' : ''
+        return (
+          <span>
+            {sa.highest_bid.display}(bid{s})
```

Comment:
Ah good catch needs a space. The `s` variable handles whether or not its bid or bids.